rpc_proxy.go

```diff
@@ -37,22 +38,20 @@ var (
 	// maxMsgRecvSize is the largest message our proxy will receive. We
 	// set this to 200MiB atm.
 	maxMsgRecvSize = grpc.MaxCallRecvMsgSize(1 * 1024 * 1024 * 200)
-	connectUrlPath = filepath.Join(
-		btcutil.AppDataDir("tdex-dashboard", false), "tdexdconnect.txt",
+	configPath     = filepath.Join(
+		btcutil.AppDataDir("tdex-dashboard", false), "config.json",
```

Comment:
maybe better calling like the reducer? 
```suggestion
		btcutil.AppDataDir("tdex-dashboard", false), "settings.json",
```